.github/ISSUE_TEMPLATE/setup-eventbrite.md

```diff
@@ -0,0 +1,112 @@
+  by lead instructor
+title: Scheduling courses with Eventbrite on [DATE-OR-SESSION-DESCRIPTION] for [INSTRUCTOR-NAME]
+labels: session-setup
+assignees: ''
+
+---
+
+The lead instructor needs to schedule events on [Eventbrite](https://www.eventbrite.com) using the [RSE
+Sheffield](https://www.eventbrite.co.uk/o/rse-sheffield-29660305889) organisation. If you have not already been setup as
+an administrator for this you should request to be included by another team member (see [RSES
+Handbook](https://github.com/RSE-Sheffield/rses-handbook/blob/master/pages/eventbrite.Rmd) for further details).
+
+**NB** - The easiest approach is to copy an existing event. From the page listing events select **Past** and click on
+the three dots to the right of a past event and select `Copy Event`. Edit the title to remove `Copy of` and update the
+dates and times.
+
+
+## Basic Info
+
+The following fields should be entered into each field on the **Basic Info** page.
+
+| Field               | Value                                                      |
+|:--------------------|------------------------------------------------------------|
+| Event Title         | Conda Environments for Effective and Reproducible Research |
+| Organiser           | RSE Sheffield                                              |
+| Type                | Class, Training, or Workshop                               |
+| Category            | Science & Technology                                       |
+| Tags                | git, github, gitkraken                                     |
+| Location            | Online Event (unless otherwise)                            |
+| Date and Time       | Single Event                                               |
+| Event Starts        | Start Date/Time                                            |
+| Event Ends          | End Date/Time                                              |
+| Time Zone           | United Kingdom Time                                        |
+| Event Page Language | English (UK)                                               |
+
+
+
+## Details
+
+A banner can be added to the event, some sample images are under the `resources/` directory of this repository, but if
+you have copied a past event this will already be in place.
+
+The following fields should be entered into each field on the **Details** page (they can be copy and pasted from a past event).
+
+### Summary
+
+
+> This is an introductory course, teaching the use of Conda Virtual Environments required to manage reproducible
+> research.
+
+### Description
+
+> How can we effectively work together on the same code?
+> This is an introductory course, teaching the use of Conda virtual environments for effective and reproducible research
+> development lifecycle. These skills are essential for reproducible research.
+> This event is for University of Sheffield researchers and research students only. Please register with an @shef.ac.uk
```

Comment:
I've started rewording this as "a University of Sheffield email address" since there are lots of staff with @amrc.uk (or something) addresses that are relevant.